BUILD

```diff
@@ -418,7 +418,9 @@ cc_test(
     copts = COPTS,
     data = [
         ":test_plugin",
-    ],
+    ] + glob([
+        "src/google/protobuf/**/*",
```

Comment:
Why do we need to include all files here? I remember I tested it before and "bazel test :protobuf-test" was passing for me.